html-css/week2/style.css

```diff
@@ -0,0 +1,190 @@
+:root {
+  --white: #ffffff;
+  --lith-gray: #eae8e8;
+  --gray: #808080;
+  --dark-gray: #454444;
+  --black: #000000;
+  --purple: #ab5cdb;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
+  color: var(--black);
+}
+
+body {
+  min-width: 20rem;
+}
+
+p {
+  color: var(--gray);
+  margin-bottom: 0.625rem;
+}
+
+h1,
+h2,
+h3 {
+  text-transform: uppercase;
+  margin-bottom: 0.625rem;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.container {
+  padding: 1.25rem 1.25rem;
+}
+
+.title_section-grey {
+  background-color: var(--lith-gray);
+}
+
+p.article-title {
+  text-transform: lowercase;
+}
+
+.header-nav__items {
+  text-transform: lowercase;
+}
+
+.header-nav__item a {
+  margin-top: 0.625rem;
+  color: inherit;
+}
+
+.header-nav__item a:hover,
+.header-nav__item a:active {
+  color: var(--gray);
+}
+
+.img-container img {
+  width: 100%;
+  height: 30%;
+}
+
+.logo-box {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+iframe {
+  margin-top: 1.25rem;
+  width: 90%;
+  height: 70%;
+}
+
+footer {
+  background-color: var(--purple);
+}
+
+.footer h3,
+.footer p,
+.footer a {
+  color: var(--white);
+}
+
+@media (min-width: 40rem) {
+  h1 {
+    font-size: 3rem;
+  }
+  h2 {
+    font-size: 2rem;
+  }
+
+  h3 {
+    font-size: 1.5rem;
+  }
+
+  .title_section-grey {
+    padding: 1.25rem 0;
+  }
+
+  a {
+    font-size: 1.25rem;
+  }
+
+  .header-nav__items {
+    display: flex;
+    font-weight: bold;
+  }
+
+  .header-nav__item {
+    margin: 1.25rem 3.125rem 1.25rem 0rem;
+  }
+
+  .img-container img {
+    margin-bottom: 2.5rem;
+  }
+
+  .article-title {
+    font-size: 1.5rem;
+  }
```

Comment:
Your `h3` also has the same `font-size`. Try to combine selectors if they have the same property and value. Save space and write less code :)